projects/media-viewer/src/lib/utils/mouse-selection-utils.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { TestBed } from '@angular/core/testing';
+import {MouseSelectionUtils} from './mouse-selection-utils';
+
+describe('Mouse Selection Utilities', () => {
+  beforeEach(() => {
```

Comment:
I believe you can remove this beforeEach step

Reply:
because its static?

Reply:
It's just a class, there is no need for an Angular dependency

Reply:
Done, can you review this aspect again.